include/seqan3/io/sequence_file/format_fasta.hpp

```diff
@@ -135,7 +135,7 @@ class format_fasta
                                id_type && id,
```

Comment:
Note that #1453 (waiting for review by you :wink) also touches these files!

Reply:
My touch is way lighter :smirk_cat: Can you benchmark after my chagne again and see if we still need the extensive changes you introduce in #1453 ?